ank-core/src/main/kotlin/io/kategory/ank/formatter.kt

```diff
@@ -0,0 +1,46 @@
+package io.kategory.ank
```

Comment:
Why `formatter.kt` is under `io.kategory.ank` package? Shouldn't be `arrow.ank`?